src/config/constants/multicall/abi.json

```diff
@@ -0,0 +1,143 @@
+[
```

Comment:
Do we now have two multicall implementations? https://github.com/pancakeswap/pancake-frontend/blob/2ac9618a63dc6fbb46911c1d4ae9d6318cd0051d/src/hooks/useContract.ts#L210 which uses `src/config/constants/multicall/abi.json`, and `0x1Ee38d535d541c55C9dae27B12edf090C608E6Fb` / `0x301907b5835a2d723Fe3e9E8C5Bc5375d5c1236A`, 

And then also:
 
https://github.com/pancakeswap/pancake-frontend/blob/7b7c12cc84a98ebfba2b0e3b43502b2a712c3033/src/utils/contractHelpers.ts#L129 (used by `src/utils/multicall.ts`) which uses `src/config/abi/Multicall.json` & `0xfF6FD90A470Aaa0c1B8A54681746b07AcdFedc9B` / `0x8F3273Fb89B075b1645095ABaC6ed17B2d4Bc576`

Presuming we need both - it would be good to distinguish this exchange-specific ABI/contract?